src/lwc/geGiftEntryFormApp/geGiftEntryFormApp.js

```diff
@@ -100,6 +100,365 @@ export default class GeGiftEntryFormApp extends NavigationMixin(LightningElement
         table.handleSubmit(event);
     }
 
+    /*******************************************************************************
```

Comment:
I think Prince's latest work on the shared branch moved these methods that handle submit down into the geFormRenderer component - does this work require that they move back up into the App component?

Reply:
No, I must have read a merge conflict wrong here and need to revert this.